pkg/cmd/roachtest/tests/mixed_version_job_compatibility_in_declarative_schema_changer.go

```diff
@@ -109,25 +109,56 @@ func executeSupportedDDLs(
 		if err := testUtils.disableJobAdoption(ctx, t.L(), nodes); err != nil {
 			return err
 		}
-		node := option.NodeListOption{helper.RandomNode(r, nodes)}
-		planAndRunSchemaChange(ctx, t, c, node, `COMMENT ON DATABASE testdb IS 'this is a database comment'`)
-		planAndRunSchemaChange(ctx, t, c, node, `COMMENT ON SCHEMA testdb.testsc IS 'this is a schema comment'`)
-		planAndRunSchemaChange(ctx, t, c, node, `COMMENT ON TABLE testdb.testsc.t IS 'this is a table comment'`)
-		planAndRunSchemaChange(ctx, t, c, node, `COMMENT ON COLUMN testdb.testsc.t.i IS 'this is a column comment'`)
-		planAndRunSchemaChange(ctx, t, c, node, `COMMENT ON INDEX testdb.testsc.t@idx IS 'this is a index comment'`)
-		planAndRunSchemaChange(ctx, t, c, node, `COMMENT ON CONSTRAINT check_j ON testdb.testsc.t IS 'this is a constraint comment'`)
-		planAndRunSchemaChange(ctx, t, c, node, `ALTER TABLE testdb.testsc.t ADD COLUMN k INT DEFAULT 35`)
-		planAndRunSchemaChange(ctx, t, c, node, `ALTER TABLE testdb.testsc.t DROP COLUMN k`)
-		planAndRunSchemaChange(ctx, t, c, node, `ALTER TABLE testdb.testsc.t2 ADD PRIMARY KEY (i)`)
-		planAndRunSchemaChange(ctx, t, c, node, `ALTER TABLE testdb.testsc.t2 ALTER PRIMARY KEY USING COLUMNS (j)`)
-		planAndRunSchemaChange(ctx, t, c, node, `DROP SEQUENCE testdb.testsc.s`)
-		planAndRunSchemaChange(ctx, t, c, node, `DROP TYPE testdb.testsc.typ`)
-		planAndRunSchemaChange(ctx, t, c, node, `DROP VIEW testdb.testsc.v`)
-		planAndRunSchemaChange(ctx, t, c, node, `DROP TABLE testdb.testsc.t`)
-		planAndRunSchemaChange(ctx, t, c, node, `DROP TABLE testdb.testsc.t2`)
-		planAndRunSchemaChange(ctx, t, c, node, `DROP SCHEMA testdb.testsc`)
-		planAndRunSchemaChange(ctx, t, c, node, `DROP DATABASE testdb CASCADE`)
 
+		// DDLs supported since V22_2.
+		v222DDLs := []string{
+			`COMMENT ON DATABASE testdb IS 'this is a database comment'`,
+			`COMMENT ON SCHEMA testdb.testsc IS 'this is a schema comment'`,
+			`COMMENT ON TABLE testdb.testsc.t IS 'this is a table comment'`,
+			`COMMENT ON COLUMN testdb.testsc.t.i IS 'this is a column comment'`,
+			`COMMENT ON INDEX testdb.testsc.t@idx IS 'this is a index comment'`,
+			`COMMENT ON CONSTRAINT check_j ON testdb.testsc.t IS 'this is a constraint comment'`,
+			`ALTER TABLE testdb.testsc.t ADD COLUMN k INT DEFAULT 35`,
+			`ALTER TABLE testdb.testsc.t DROP COLUMN k`,
+			`ALTER TABLE testdb.testsc.t2 ADD PRIMARY KEY (i)`,
+			`ALTER TABLE testdb.testsc.t2 ALTER PRIMARY KEY USING COLUMNS (j)`,
+		}
+
+		// DDLs supported since V23_1.
+		v231DDls := []string{
```

Comment:
I think we also supported the following in V23_1 in DSC. Can you also include them here?
- `ADD FOREIGN KEY`
- `ADD FOREIGN KEY NOT VALID`
- `ADD UNIQUE WITHOUT INDEX`
- `ADD UNIQUE WITHOUT INDEX NOT VALID`
- `VALIDATE CONSTRAINT`
- `DROP CONSTRAINT`